inference-engine/src/legacy_api/include/legacy/ie_layers.h

```diff
@@ -1536,6 +1536,76 @@ class INFERENCE_ENGINE_INTERNAL_CNNLAYER_CLASS(PadLayer): public CNNLayer {
     ~PadLayer() override;
 };
 
+/**
+ * @deprecated Migrate to IR v10 and work with ngraph::Function directly. The method will be removed in 2021.1
```

Comment:
@andrejsokolov @DariaMityagina 2021.1 is out already. Please implement interpolate without adding new deprecated class

Reply:
Okay, thanks.